.github/workflows/run_test_cases.yaml

```diff
@@ -77,6 +77,7 @@ jobs:
             make ensure-rebar3
             # fetch all deps and compile
             make ${{ matrix.profile }}
+            make static_checks
```

Comment:
I think having PLT cache here would be great, otherwise the CI feedback might be delayed noticeably. Also wondering why not separate workflow?